css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,18 @@
+button {
+  font-size: 20px;
+}
+
+button:hover {
+  color: white;
+  background-color: black;
+  cursor: pointer;
+}
+
+tr:nth-child(odd) {
```

Comment:
For future references, try using table.striped thead > tr & even table.striped tbody > tr:nth-child().